resources/lib/utils.py

```diff
@@ -158,7 +138,7 @@ def url_to_episode(url):
     if url.startswith('//www.vrt.be/vrtnu/a-z/'):
         # medium episode url
         return url.replace('//www.vrt.be/vrtnu/a-z/', '/vrtnu/a-z/')
-    if url.startswith('/vrtnu/a-z/'):
+    if url.startswith('/vrtnu/a-z/') or url.startswith('/vrtmax/a-z/'):
```

Comment:
Used or instead of dropping 'vrtnu' since this would be a BC break for users with local HTTP cache.